docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../auth/scim/
+  - ../../enterprise/scim/
+description: Learn how to use SCIM provisioning to synchronize users and groups from your identity provider to Grafana.
```

Comment:
Consider adding a bit more description here. Generally aim for at least 150 characters (it's currently 106).

https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases